components/expenses/ExpenseFormPayeeInviteNewStep.js

```diff
@@ -0,0 +1,322 @@
+  },
+});
+
+const ExpenseFormPayeeInviteNewStep = ({ formik, collective, onCancel, onNext }) => {
+  const intl = useIntl();
+  const { formatMessage } = intl;
+  const { values, errors } = formik;
+  const stepOneCompleted = values.payee?.name && values.payee?.email;
+
+  const setPayoutMethod = React.useCallback(({ value }) => formik.setFieldValue('payoutMethod', value), []);
+  const [showAdditionalInfo, setAdditionalInfo] = React.useState(false);
+
+  return (
+    <Fragment>
+      <Grid
+        gridTemplateColumns={['100%', 'calc(50% - 8px) calc(50% - 8px)']}
+        gridColumnGap={[null, 2, null, 3]}
+        gridAutoFlow="dense"
+      >
+        <Box>
+          <Field name="payee.name">
+            {({ field }) => (
+              <StyledInputField name={field.name} label={formatMessage(msg.nameLabel)} labelFontSize="13px" mt={3}>
+                {inputProps => <StyledInput {...inputProps} {...field} />}
+              </StyledInputField>
+            )}
+          </Field>
+        </Box>
+        <Box>
+          <Field name="payee.email" required>
+            {({ field }) => (
+              <StyledInputField
+                name={field.name}
+                label={formatMessage(msg.emailTitle)}
+                labelFontSize="13px"
+                error={errors.payee?.email}
+                mt={3}
+              >
+                {inputProps => <StyledInput {...inputProps} {...field} type="email" />}
+              </StyledInputField>
+            )}
+          </Field>
+        </Box>
+
+        {!showAdditionalInfo ? (
+          <Box gridColumn={[null, '1 / span 2']} mt={3}>
+            <MessageBox type="info" fontSize="12px">
+              {formatMessage(msg.additionalInfo)}
+              <br />
+              {formatMessage(msg.additionalInfoBtn, {
+                Link: getI18nLink({ onClick: () => setAdditionalInfo(true) }),
+              })}
+            </MessageBox>
+          </Box>
+        ) : (
+          <Fragment>
+            <Box>
+              <FastField name="payeeLocation.country">
+                {({ field }) => (
+                  <StyledInputField
+                    name={field.name}
+                    label={formatMessage(msg.country)}
+                    labelFontSize="13px"
+                    error={formatFormErrorMessage(intl, errors.payeeLocation?.country)}
+                    required
+                    mt={3}
+                  >
+                    {({ id, error }) => (
+                      <InputTypeCountry
+                        data-cy="payee-country"
+                        inputId={id}
+                        onChange={value => formik.setFieldValue(field.name, value)}
+                        value={field.value}
+                        error={error}
+                      />
+                    )}
+                  </StyledInputField>
+                )}
+              </FastField>
+              <I18nAddressFields
```

Comment:
I think for this `I18nAddressFields` the `onChange` prop is marked as required. Better to put a void function for that instead of not specifying it as whenever this component is called the developer console will print that warning. 😉